static/js/components/nl_feedback.tsx

```diff
@@ -33,117 +34,86 @@ interface NlChartFeedbackPropType {
   id: string;
 }
 
+const FEEDBACK_OPTIONS = [
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.THUMBS_UP,
+    content: <>&#128077;</>,
+    tooltip: "The chart is relevant for the query",
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.THUMBS_DOWN,
+    content: <>&#128078;</>,
+    tooltip: "The chart is not relevant for the query",
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.PROMOTE,
+    content: <>&#11014;</>,
+    tooltip: "The chart should be promoted up",
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.DEMOTE,
+    content: <>&#11015;</>,
+    tooltip: "The chart should be demoted",
+  },
+  {
+    sentiment: CHART_FEEDBACK_SENTIMENT.FACE_PALM,
+    content: <>&#129318;&#127995;</>,
+    tooltip: "The chart is embarassing or inappropriate for the the query",
```

Comment:
nit: typo

Reply:
good catch- fixed!

Reply:
Hmm the checked in code has `the the` - https://github.com/datacommonsorg/website/blob/master/static/js/components/nl_feedback.tsx#L61 ?